CONTRIBUTING.md

```diff
@@ -57,6 +57,75 @@ You can build and start a local docs website by running these two commands.
   - `npm run docs-buildDockerImage` OR `docker build -t squidfunk/mkdocs-material ./docs/`
   - `npm run docs-runLocalDocker` OR `docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material`
 
+### Tests
+
+Tests are under `tests` folder of each modules and splitted into two categories: Unit tests and e2e tests.
+
+This split happen thanks to [jest-runner-groups](https://www.npmjs.com/package/jest-runner-groups).
+
+Unit tests, under `tests/unit` folder are standard jest tests.
+
+Integration tests, under `tests/e2e` folder, will test the module features by deploying lambdas into your AWS Account (thanks to CDK lib for typescript) and use aws sdk to invoke them and assert on expected behavior. All of it orchestrated using standard Jest framework. Since it's deploying infrastructure, it will need an AWS account.
```

Comment:
```suggestion
Integration tests under `tests/e2e` folder on the other hand will test the module features by deploying Lambdas functions into your AWS Account. Infrastructure deployment is possible thanks to CDK lib for Typescript and `aws sdk` is used to invoke the functions and assert on the expected behavior. All of this is orchestrated using the same Jest framework used for the unit tests.

Since running integration tests will deploy AWS resources you will need an AWS account and might incur in some costs which should be covered by the [AWS Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=*all&awsf.Free%20Tier%20Categories=*all). If you already don't have an AWS Account follow [these instructions to create one](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).

When contributing to this repository these integration tests are run by the maintainers before merging a PR.
```